.github/workflows/main.yml

```diff
@@ -113,6 +113,27 @@ jobs:
             .ci/check-format.sh
       shell: bash
   
+  clang-static-analysis:
```

Comment:
Set the name `static-analysis`. Later, we can append more tools in this job.

Reply:
Fixed as suggested. 

Additional comment is added.